lib/main_common.pm

```diff
@@ -597,7 +597,14 @@ sub load_jeos_openstack_tests {
     } else {
         loadtest 'publiccloud/ssh_interactive_start', run_args => $args;
     }
-    #    loadtest "jeos/image_info";
+
+    if (get_var('CI_VERIFICATION')) {
+        loadtest 'jeos/verify_cloudinit', run_args => $args;
+        loadtest("publiccloud/ssh_interactive_end", run_args => $args);
```

Comment:
I'm confused why you add this here. If we don't wanna run things on OpenStack, we can even remove this and ssh_start, can't we?

Reply:
The test is using multiple consoles, so I have kinda found it useful to run interactive ssh. As it should run in a single test suite, we need to close the tunnel here. Or do you prefer to run some additional tests?

Reply:
But ssh_interactive_start/end is used for PublicCloud/OpenStack tests. We are not planning to run this test on those platforms, right?